### PR TITLE
Add support for vxfs and zfs filesystem types

### PR DESCRIPTION
**Pull request by ned21**

### CODE REVIEW

**Comment by jrha** [comment]
LGTM, travis failed on linting errors, but nothing introduced by this change.

File: quattor/filesystems.pan [hunk 1]
```diff
@@ -39,7 +39,7 @@ type structure_filesystem = {
 	"mount"		: boolean
 	"mountopts"	: string = "defaults" # "Mount options"
 	"type"		: string with match (SELF,
-			     "^(ext[2-4]|reiserfs|reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|none)$")
+			     "^(ext[2-4]|reiserfs|reiser4|xfs|swap|vfat|jfs|ntfs|tmpfs|vxfs|zfs|none)$")
```
Comment (stdweird) [inline quattor/filesystems.pan]: non-blocking question: does this deserve it's own type?

Comment (jrha) [inline quattor/filesystems.pan]: Does anything else refer to a filesystem?

**Comment by ned21** [comment]
I cleaned up the whitespace in the type I was changing - let's see if that gets us a bit closer.

**Pull request merged**